src/language/lang_es.php

```diff
@@ -0,0 +1,643 @@
+    'Extrayendo',
+  //Translations for Utils.php's time_ago() function
+  'just now' =>
+    'justo ahora',
+  'd' =>
+    'd', //day
+  'hr' =>
+    'hr', //hour
+  'min' =>
+    'min', //minute
+  'sec' =>
+    'seg', //second
+  'ds' =>
+    'd', //days
+  'hrs' =>
+    'hrs', //hours
+  'mins' =>
+    'mins', //minutes
+  'secs' =>
+    'segs', //seconds
+  'ago' =>
+    'atrás',
+  //Translations for ModalControllers
+  'begin_' =>
+    'comenzar_',
+  'Are you sure you want to kick off the game? Logs will be cleared and progressive scoreboard will start' =>
+    '¿Está seguro de que desea iniciar el juego? Los logs se borrarán y el panel de puntuaciones iniciará',
+  'end_' =>
+    'finalizar_',
+  'Are you sure you want to finish the current game?' =>
+    'Está seguro que desea finalizar el juego actual?',
+  'Are you sure you want to logout from the game?' =>
+    'Está seguro que desea salir del juego?',
+  'Saved' =>
+    'Guardado',
+  'All changes have been successfully saved.' =>
+    'Todos los cambios fueron guardados exitosamente.',
+  'Error' =>
+    'Error',
+  'Sorry your form was not saved. Please correct the all errors and save again.' =>
+    'Lo sentimos tu formulario no fue guardado. Porfavor corrige los errores y guarda nuevamente.',
+  'cancel_' =>
+    'cancelar_',
+  'Are you sure you want to cancel? You have unsaved changes that will be reverted.' =>
+    'Está seguro que sea cancelar? Tienes guardos sin guardad que serán revertidos.',
+  'choose_logo' =>
+    'elegir_logo',
+  'captured_' =>
+    'captured_',
+  'flag_owner_' =>
+    'flag_owner_',
+  'INACTIVE' =>
+    'INACTIVO',
+  'PTS' =>
+    'PTS',
+  'category' =>
+    'categoría',
+  'capture_' =>
+    'capture_',
+  'Insert your answer' =>
+    'Introduce tu respuesta',
+  'Request Hint' =>
+    'Solicitar Pista',
+  'Submit' =>
+    'Enviar',
+  'hint_' =>
+    'pista_',
+  'first_capture' =>
+    'first_capture',
+  'completed_by' =>
+    'completado_por',
+  'scoreboard_' =>
+    'scoreboard_',
+  'filter_' =>
+    'filter_',
+  'rank_' =>
+    'rank_',
+  'team_name_' =>
+    'team_name_',
+  'quiz_pts_' =>
+    'quiz_pts_',
+  'flag_pts_' =>
+    'flag_pts_',
+  'base_pts_' =>
+    'base_pts_',
+  'total_pts_' =>
+    'total_pts_',
+  'team_' =>
+    'team_',
+  'team_members' =>
+    'team_members',
+  'base_pts' =>
+    'base_pts',
+  'quiz_pts' =>
+    'quiz_pts',
+  'flag_pts' =>
+    'flag_pts',
+  'total_pts' =>
+    'total_pts',
+  'Tool bars are located on all edges of the gameboard. Tap a category to expand and close each tool bar.' =>
+    'Las barras de herramientas se encuentran en todos los bordes del panel de juego. Pulsa en una categoría para ampliar y cerrar cada barra de herramientas.',
+  'Tool_Bars' =>
+    'Tool_Bars',
+  'Tap the "Game Clock" to keep track of time during gameplay. Don’t let time get the best of you.' =>
+    'Pulsa el "Reloj del Juego" para hacer seguimiento del tiempo. No dejes que el tiempo te gane.',
```

Comment:
If we are not modifying the label for the clock, we need to keep the text here:
"Pulsa el "Game Clock" para hacer seguimiento del tiempo. No dejes que el tiempo te gane."